db/migrate/20180507134810_azure_normalize_image_name.rb

```diff
@@ -0,0 +1,12 @@
+class AzureNormalizeImageName < ActiveRecord::Migration[5.0]
+  class Vm < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time("Updating Azure template name") do
+      Vm.where(:type => "ManageIQ::Providers::Azure::CloudManager::Template")
```

Comment:
Should this be `VmOrTemplate.where(...)`?

Reply:
Works either way afaik, since I'm specifying the type.

Reply:
I think templates will not be returned by `Vm.where()`, I think it requires `VmOrTemplate`. Can you check on the console?